pkgs/by-name/si/sillytavern/sillytavern.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Icon=sillytavern
+Name=SillyTavern
+Exec=sillytavern
+Terminal=true
+Hidden=false
+Categories=Education
```

Comment:
Is this the better category for this package?

Reply:
I would say go with `Network;Chat`, where chat is listed as an [Additional Category](https://specifications.freedesktop.org/menu-spec/latest/apas02.html) in the freedesktop spec.

Reply:
For easy committing:

```suggestion
Categories=Network;Chat
```